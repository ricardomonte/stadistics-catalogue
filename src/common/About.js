import React from 'react';
import GameStyles from '../styles/About.module.css';

const About = () => (
  <>
    <div className={GameStyles.about} />
    <div className={GameStyles.aboutText}>
      <h1 className={GameStyles.title}>About</h1>
      <p className={GameStyles.aboutInfo}>
        FGPG comes from From Gamers To Gamers we look
        <br />
        We look for the top games on the market, and we give you the details of
        them.
        <br />
        We dream that everyone can know the games that everyone talks about.
      </p>
    </div>
  </>
);

export default About;
