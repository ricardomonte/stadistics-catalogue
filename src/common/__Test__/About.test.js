import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import About from '../About';

test('Content from About matches snapshot', () => {
  const content = render(<About />);
  expect(content.container).toMatchSnapshot();
  expect(screen.getByText('About')).toBeInTheDocument();
});
