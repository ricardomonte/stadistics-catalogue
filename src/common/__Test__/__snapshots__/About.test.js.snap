// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content from About matches snapshot 1`] = `
<div>
  <div
    class="about"
  />
  <div
    class="aboutText"
  >
    <h1
      class="title"
    >
      About
    </h1>
    <p
      class="aboutInfo"
    >
      FGPG comes from From Gamers To Gamers we look
      <br />
      We look for the top games on the market, and we give you the details of them.
      <br />
      We dream that everyone can know the games that everyone talks about.
    </p>
  </div>
</div>
`;
